packages/proto/gozd/v1/git_ops.proto

```
@@ -275,3 +275,61 @@ message GitWorktreeRemoveRequest {
+// `git log -L` は patch が出るのが default だが、popover では commit 一覧だけ欲しいため
+// `--no-patch` で抑制する。
+message GitLogLineRequest {
+  string dir = 1;
+  string rel_path = 2;
+  string rev = 3;
+  uint32 line = 4;
+  uint32 max_count = 5;
+}
+
+message GitLogLineResponse {
+  repeated GitCommit commits = 1;
```

Buf (1.69.0)

[error] cannot find `GitCommit` in this scope

(COMPILE)